SearchBar/AutoCompleteHandler.js

```diff
@@ -9,4 +10,22 @@ export default class AutoCompleteHandler extends GridDataAutoCompleteHandler {
         var result = super.needOperators(parsedCategory);
         return this.scroll ? ["==", "!="] : result.concat(["startsWith"]);
     }
+
+    needValues(parsedCategory, parsedOperator) {
+        const found = _.find(this.options, f => f.columField == parsedCategory || f.columnText == parsedCategory);
+        const parsedField = found ? found.columField : null;
```

Comment:
above this statement you should do 
```
if (found === null) {
     return []
}
```
And then remove the check on if (found != null)